recipes/bob.extension/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+PYTHONDONTWRITEBYTECODE=1 $PYTHON setup.py install
```

Comment:
When `setuptool` is used we recommend adding those options to be able to properly uninstall software later on. (Basically it makes the installation play nice with others.)